lib/identity/cookie_coder.rb

```diff
@@ -0,0 +1,58 @@
+require "base64"
+
+module Identity
+  # CookieCoder encodes and decodes payload before setting it to cookie
+  # Different from FernetCookieCoder, it doesn't use Ruby Marshal.
+  # The encrypted cookie is meant to consumed by other programming languages other than Ruby.
+  class CookieCoder
```

Comment:
The intent of `CookieCoder` is to encode/decode a hash for setting in the cookie. The name is to the point IMO.